### PR TITLE
new data hub

### PR DESCRIPTION
**Pull request by JeremyLikness**
Initial layout for new data hub. Addresses #3833

### CODE REVIEW

File: entity-framework/dotnet-data/index.yml
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:81]: The intent is for people new to .NET/etc. They won't search for Cosmos, but rather "document database" and the one we happen to have is Cosmos.

File: entity-framework/dotnet-data/index.yml
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:75]: that page has both Azure and on-premises docs

File: entity-framework/dotnet-data/index.yml
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:95]: nice I'll add it

File: entity-framework/dotnet-data/index.yml
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:115]: great point! I'm mixed thoughts. To me discoverability is the most important and when looking at databases, seems not like a bad thing to include "this is our recommended way to connect to databases (in most scenarios).

File: entity-framework/dotnet-data/index.yml
Comment (JeremyLikness) [reply entity-framework/dotnet-data/index.yml:194]: yes, it's one of the preferred ways in .NET to send/receive realtime/streaming data

File: entity-framework/dotnet-data/index.yml
Comment (bricelam) [inline entity-framework/dotnet-data/index.yml:8]: Can we style it *.NET Data*? (everywhere) This just makes me ask, "What's data documentation?"

**Review by ajcvickers — APPROVED** [review]
This looks great! :shipit:

**Comment by JeremyLikness** [comment]
If @smitpatel is able to take a quick look, I will merge after his approval.

**Review by luisquintanilla — APPROVED** [review]
This is great @JeremyLikness. LGTM :shipit:

**Pull request merged**
